.ci/kitchen-macosxhighsierra-py3-pytest

```diff
@@ -0,0 +1,22 @@
+@Library('salt@master-1.7') _
+
+// Pre-nox pipeline
+runTestSuite(
+    concurrent_builds: 2,
+    distro_name: 'macosx',
+    distro_version: 'highsierra',
+    env: env,
+    golden_images_branch: 'master',
+    jenkins_slave_label: 'kitchen-slave-mac',
+    kitchen_platforms_file: '/var/jenkins/workspace/pre-golden-nox-platforms.yml',
+    kitchen_verifier_file: '/var/jenkins/workspace/nox-verifier.yml',
```

Comment:
this is the default so we don't have to define it right?

Reply:
Hmm, yes?